src/main/java/net/openhft/chronicle/network/connection/ThreadPrivateTLongObjHashMap.java

```diff
@@ -34,11 +34,13 @@
 
 /**
  * Class to determine if several threads are using a TLongObjectMap.
- *
+ * <p>
  * This class should not be used in production code.
  *
  * @param <V> value type
+ * @deprecated To be removed in x.25 with no replacement
  */
+@Deprecated(/* For removal in x.25 */)
 public final class ThreadPrivateTLongObjHashMap<V> implements TLongObjectMap<V> {
```

Comment:
This looks like some kind of test utility. It's not used anywhere and seems like it shouldn't live in network in any case